src/credentials-manager/index.js

```diff
@@ -83,13 +84,15 @@ class CredentialsManager {
    * @param {String} description Android Only - optional - the text to use as description in the authentication screen. On some Android versions it might not be shown. Passing null will result in using the OS's default value.
    * @param {String} cancelTitle iOS Only - optional - the cancel message to display on the local authentication prompt.
    * @param {String} fallbackTitle iOS Only - optional - the fallback message to display on the local authentication prompt after a failed match.
+   * @param {String} localAuthenticationStrategy iOS Only - optional - the evaluation policy to use when accessing the credentials. Defaults to LocalAuthenticationStrategy.deviceOwnerWithBiometrics.
    * @returns {Promise}
    */
   async requireLocalAuthentication(
     title,
     description,
     cancelTitle,
     fallbackTitle,
+    localAuthenticationStrategy = LocalAuthenticationStrategy.deviceOwnerWithBiometrics,
```

Comment:
```suggestion
   * @param {String} strategy iOS Only - optional - the evaluation policy to use when accessing the credentials. Defaults to LocalAuthenticationStrategy.deviceOwnerWithBiometrics.
   * @returns {Promise}
   */
  async requireLocalAuthentication(
    title,
    description,
    cancelTitle,
    fallbackTitle,
    strategy = LocalAuthenticationStrategy.deviceOwnerWithBiometrics,
```
Inside the `requireLocalAuthentication()` method, prepending arguments with  `localAuthentication` seems a bit redundant.